source/function.cpp

```diff
@@ -296,8 +302,18 @@ bool is_skipping_requested(FunctionDecl const *F, Config const &config)
 	}
 	//outs() << "OK\n";
 
-	for(auto & t : get_type_dependencies(F) ) skip |= is_skipping_requested(t, config);
-
+	for(auto & t : get_type_dependencies(F) ) {
+		bool dep_skip = is_skipping_requested(t, config);
+		bool primitive = is_primitive(t, config);
+		if ( primitive ) dep_skip = false;
+		skip |= dep_skip;
+	}
```

Comment:
again: could you please elaborate why do we need to take `primitive` into account?

Reply:
Hum, I was borrowing the `+primitive` terminology from shiboken, the pyside binding generator. In the light of your code pointer, renaming this to `+builtin` should be nicer, I guess.

Reply:
I guess my previous comment might have been a bit cryptic, what i meant ask is this: why do we need to check if type `t` is build-in? I think we do not since purpose of this function is to determent if binding of given function was explicitly disabled by user. Am i missing something here? If so could you please elaborate? Thanks,